code/modules/mob/living/carbon/superior_animal/attack.dm

```diff
@@ -156,7 +156,7 @@
 					if (possible_target)
 						if (possible_target == target)
 							continue //we made the concious choice to attack them
-						else if (!(prob(do_friendly_fire_chance)) && (((!attack_same && (possible_target.faction == faction)) || (possible_target in friends)) || (possible_target.friendly_to_colony && friendly_to_colony)))
+						else if (!(prob(do_friendly_fire_chance)) && !friendly_to_colony && (((!attack_same && (possible_target.faction == faction)) || (possible_target in friends))))
```

Comment:
Dosnt this mean that golems (church ones) and friendly mobs dont get shot completely?

Reply:
resolved in DM's